### DISCUSSION TITLE
Support for token and managed identity support

### DISCUSSION THREAD

**Discussion by MuhammadPathan** [sfu-db/connector-x#641] [Ideas]
feature request- Adding support to use token based auth for connecting into SQL Server so for example being able to use Azure Managed  Identity

**Comment by HWiese1980** [comment]
Yeah, or AWS IAM. Generally, AWS supports using the AWS API generated token as password for the corresponding user. That's how it authenticates against the database, using a temporary password (token) generated by IAM. However, it would be nice to have that somehow integrated into connectorx, so that there is no need to call the AWS API explicitly. Would make the user code a lot cleaner.
